dj_secure_settings/loader.py

```diff
@@ -52,10 +51,13 @@ def load_secure_settings(project_name=None, environment=None):
     try:
         _load_params_from_ssm(config, f'/{env}/defaults/')
         _load_params_from_ssm(config, f'/{env}/{project_name}/')
-    except ExpiredTokenException as e:
-        # could not load params from Parameter Store, but that may be ok
-        logging.debug("Couldn't load params from SSM: missing or expired token")
-        pass
+    except ClientError as e:
```

Comment:
This `ExpiredTokenException` actually seems to have referred to `STS.Client.exceptions.ExpiredTokenException`, which isn't imported, so this was throwing an import error.

It turns out that the `STS` client exception isn't directly capturable anyway -- instead, boto3 throws `botocore.exceptions.ClientError`, which exposes the underlying cause in its string representation: `str(e) == "An error occurred (ExpiredTokenException) when calling the GetParametersByPath operation: The security token included in the request is expired"`.